### ISSUE TITLE
Can't play the game with this mod installed

### ISSUE THREAD

**Issue by LOLC2k** [BarleyFlour/RespecMod#49]
I have 1.05.02 installed. When I have this version installed, the game itself will fail to load (gives an error about halfway through the progress bar, before the main menu screen). I see that on my mod manager it says there's an update (2.6.1). If I update to that version, the game does load, but of course the respec button doesn't do anything in this version.

**Comment by Mira132** [comment]
Yeah, update 1.0.2e of the game prevents the game from loading with this mod installed.

**Comment by BarleyFlour** [comment]
Send your logs please.

**Comment by Taldren76** [comment]
I loaded 1.0.2e, but there was a long delay before the game first loaded. After that it was fine.
